Dockerfile

```diff
@@ -1,18 +1,7 @@
-FROM vaijab/nodejs:0.12.7
-
-RUN dnf install -y -q git
+FROM quay.io/ukhomeofficedigital/nodejs:v3.0.0
```

Comment:
that's the semver version of this repo, as opposed to the node version.  happy to make it just the same as the node version though